db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20181210164829) do
+ActiveRecord::Schema.define(version: 2019_01_02_161442) do
```

Comment:
Minor and unrelated, but I would let ActiveRecord do what it does and not listen to rubocop on this one (I thought db/schema.rb was an ignored file anyway)

Reply:
huh, I did't change this manually :-) Seems like new active record generates it like this?